scene/gui/box_container.h

```diff
@@ -73,7 +79,7 @@ class HBoxContainer : public BoxContainer {
 
 public:
 	HBoxContainer() :
-			BoxContainer(false) {}
+			BoxContainer(false) { is_fixed = true; }
```

Comment:
```suggestion
			BoxContainer(false),
			is_fixed(true) {}
```
Same for others.

Reply:
Does not work, I get `C2614 illegal member initialization: 'is_fixed' is not a base or member` error :/

Reply:
Ah that's weird. I guess it's fine as is then.